meta/main.yml

```diff
@@ -0,0 +1,53 @@
+galaxy_info:
```

Comment:
Does galaxy automatically import roles or is there another step in order to get them posted? They may also have a 'quality' scan we can use somewhere.

Reply:
...researching...

Reply:
If you register with the Ansible Galaxy site, you can [share roles](https://docs.ansible.com/ansible/2.7/reference_appendices/galaxy.html#import-a-role) you created. You can also set up an [integration with Travis](https://docs.ansible.com/ansible/2.7/reference_appendices/galaxy.html#travis-integrations) to automatically trigger an import in Galaxy.

Reply:
Ok cool, I guess we can test the process but maybe we can hit the webhook from CircleCI with the same result.

Reply:
On the code-quality front it seems they're also using yamllint with a specific [configuration](https://github.com/ansible/galaxy/blob/devel/galaxy/importer/linters/yamllint.yaml) that we should probably adopt.